crates/evm/core/src/backend/mod.rs

```diff
@@ -583,80 +578,12 @@ impl Backend {
         self.inner.has_snapshot_failure = has_snapshot_failure
     }
 
-    /// Checks if the test contract associated with this backend failed, See
-    /// [Self::is_failed_test_contract]
-    pub fn is_failed(&self) -> bool {
-        self.has_snapshot_failure() ||
-            self.test_contract_address()
-                .map(|addr| self.is_failed_test_contract(addr))
-                .unwrap_or_default()
-    }
-
-    /// Checks if the given test function failed
-    ///
-    /// DSTest will not revert inside its `assertEq`-like functions which allows
-    /// to test multiple assertions in 1 test function while also preserving logs.
-    /// Instead, it stores whether an `assert` failed in a boolean variable that we can read
-    pub fn is_failed_test_contract(&self, address: Address) -> bool {
-        /*
-        contract DSTest {
-            bool public IS_TEST = true;
-            // slot 0 offset 1 => second byte of slot0
-            bool private _failed;
-        }
-        */
-        let value = self.storage_ref(address, U256::ZERO).unwrap_or_default();
```

Comment:
1. this is wrong
2. the location of the failed variable is not specified and should not be assumed